cmd/link.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/manifoldco/manifold-cli/api"
+	"github.com/manifoldco/manifold-cli/config"
+	"github.com/manifoldco/manifold-cli/middleware"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	linkCmd := cli.Command{
+		Name:     "link",
+		Usage:    "Link your an account to a third-party account",
```

Comment:
Since it's very specifically github right now, we should label it as such so it's easy to follow in Help. (also has a typo in it).

> Change your account to authenticate with Github